sources/engine/Xenko.Games/OpenTK/GameWindowOpenTK.cs

```diff
@@ -257,6 +257,14 @@ public override bool IsMinimized
             }
         }
 
+        public override bool Focused
```

Comment:
This one would be nicer as `public override bool Focused => gameForm.Focused;`

Reply:
Never mind, rest of file is not... we can do that in a separate major style PR later.